src/ImageSharp/Formats/Png/PngMetaData.cs

```diff
@@ -42,6 +44,31 @@ private PngMetaData(PngMetaData other)
         /// </summary>
         public float Gamma { get; set; }
 
+        /// <summary>
+        /// Gets or sets the Rgb 24 transparent color. This represents any color in an 8 bit Rgb24 encoded png that should be transparent
+        /// </summary>
+        public Rgb24? Rgb24Trans { get; set; }
```

Comment:
I think we should not use abbrevations in public API-s. `TransparentRgb24`? @JimBobSquarePants any better idea?

Reply:
Fixed